elao.app/.manala/github/docker-compose.yaml.tmpl

```diff
@@ -0,0 +1,69 @@
+version: '3.8'
+services:
+
+  registry:
```

Comment:
Note: maybe this should goes into the action itself only, as it's used internally for cache.

Reply:
Wu could see that later :)